modules/cms/twig/Extension.php

```diff
@@ -166,6 +170,24 @@ public function themeFilter($url): string
         return $this->controller->themeUrl($url);
     }
 
+    /**
+     * Generates Vite tags via Laravel's Vite Object.
+     *
+     * @param array $arguments The list of entry points for Vite
+     * @param string $package The package name of the plugin or theme
+     * @return \Illuminate\Support\HtmlString
```

Comment:
No need to double-up @return doc with return type definition, unless you want to specify some additional content for the return doc.
```suggestion
```